js/gc-smart-data.js

```diff
@@ -336,6 +336,9 @@ window.GC = window.GC || {};
 
         if (boneAgeList && boneAgeList.length) {
             $.each(boneAgeList, function(i, o) {
+                if (o.hasOwnProperty("boneAgeMos") && o.hasOwnProperty("date")) {
```

Comment:
I'd check if the `o` object is valid first.

```
if (o && o.hasOwnProperty...
```

Reply:
`o` is object in an array and since we are doing $.each we should always have `o` .